pkg/porter/dependencies.go

```diff
@@ -16,7 +16,7 @@ type dependencyExecutioner struct {
 	*context.Context
 	Resolver BundleResolver
 	CNAB     CNABProvider
-	Manifest *config.Manifest
+	Manifest *config.RuntimeManifest
```

Comment:
I wonder if we'd want to change this `Manifest` field name to `RuntimeManifest`?

Reply:
I didn't rename this field just yet because it would have touched a _bunch_ of files and made it hard to really tell what I was thinking about changing.